prisma/schema.prisma

```diff
@@ -31,11 +31,27 @@ model User {
 }
 
 model Tenant {
-  id                  String   @id @default(uuid())
-  name                String
-  slug                String   @unique
-  createdAt           DateTime @default(now())
-  updatedAt           DateTime @updatedAt
-  clerkOrganizationId String   @unique
+  id                  String        @id @default(uuid())
+  clerkOrganizationId String        @unique
   users               User[]
+  tenantConfig        TenantConfig?
+
+  name String
+  slug String @unique
+
+  createdAt DateTime @default(now())
+  updatedAt DateTime @updatedAt
 }
+
+model TenantConfig {
```

Comment:
Not a fan of calling this TenantConfig, while it contain twillio specific fields, if you have a new integration like sendgrid, will you just add a new column here? It doesn't feel scalable, I think you could either rename this to TwillioTenantConfig, if you don't wanna deal with it now or have a generic way to store integration credentials. Like have two column, integrationName, integrationValues (JSON column)

Reply:
and use Zod for validation